### PR TITLE
Rename some methods in AlluxioMasterProcess

### PR DESCRIPTION
**Pull request by elega**
### What changes are proposed in this pull request?

Some trivial refactoring on AlluxioMasterProcess.

### Why are the changes needed?

Because we want to make some extensions to AlluxioMasterProcess in the future. Refactoring is put up as a individual PR for the ease of reviewing. 

### Does this PR introduce any user facing changes?

N/A

### CODE REVIEW

File: core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java [hunk 1]
```diff
@@ -266,7 +266,7 @@ private boolean gainPrimacy() throws Exception {
     mServingThread = new Thread(() -> {
       try {
         startCommonServices();
-        startLeaderServing(" (gained leadership)", " (lost leadership)");
+        startMasterServing(" (gained leadership)", " (lost leadership)");
```
Comment (elega) [inline core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java:269]: rename startLeaderServing -> startMasterServing

File: core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java [hunk 2]
```diff
@@ -288,7 +288,7 @@ private boolean gainPrimacy() throws Exception {
   private void losePrimacy() throws Exception {
     LOG.info("Losing the leadership.");
     if (mServingThread != null) {
-      stopLeaderServing();
+      stopServingGrpc();
```
Comment (elega) [inline core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java:291]: rename stopLeaderServing  to stopServingGrpc as it doesn't STOP the leader serving, but only stops the grpc server.

**Review by jiacheliu3 — APPROVED** [review]
LGTM, thanks :)

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**